canonical-web-and-design/ubuntu.com
# Build certs search results page

## Done

- Remove redirect
- Build search results page for `/certifications`
- Nice to have - when selecting the filters the list automatically updates. This isn't on the current site so would be an additional feature maybe looked at once the other pages are built. 


## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/certification
- Use the drop down and search bar to search for hardware. 
- Try selecting different filters
- Check all links work correctly


## Issue / Card

Fixes [#146](https://github.com/canonical-web-and-design/certification.ubuntu.com/issues/146)
